exercises/practice/difference-of-squares/.meta/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "authors": ["gvrooyen"],
```

Comment:
Can we run `configlet fmt` across these exercises? In the future, `configlet create --practice-exercise` can be used to create the files for a practice exercise and correctly formatted. The exercise generation script can incorporate that command and then append the content needed.

Reply:
Done